shared/src/commonMain/kotlin/io.newm.shared/internal/api/models/Users.kt

```diff
@@ -9,7 +9,7 @@ internal data class LogInUser(
 )
 
 @Serializable
-internal data class GoogleSignInRequest(val accessToken: String)
+expect class GoogleSignInToken
```

Comment:
I see why the class is name Users, should we move this to it's own class?

Reply:
This should be `internal` since it's in the API layer.